lib/sql-string.js

```diff
@@ -61,6 +61,10 @@ SqlString.escape = function(val, stringifyObjects, timeZone, dialect, field) {
     return val + '';
   }
 
+  if (field && field.type && field.type.validate) {
+    field.type.validate(val);
```

Comment:
@fixe You mean all non instance-create/update queries? Not sure, instance-validator wouldn't be enough certainly.